aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -1,23 +1,53 @@
 ---
-title: ASP.NET Core Module configuration reference
+title: ASP.NET Core Module
```

Comment:
This doc looks unwieldy now.  Wouldn't the conceptual overview material fit into the fundamentals/severs article?